students/EricAdams/timing_context_manager/generator_timer_context_manager_write_to_file_obj.py

```diff
@@ -0,0 +1,35 @@
+#! /usr/bin/env python
+'''
+Implement a simple timer context manager as a generator to take a file
+object and write to it. Wrap it all in the contextlib.contextmanager
+'''
+import time
+import sys
+from contextlib import contextmanager
+
+
+class Timer:
+    def __init__(self, file_object=sys.stdout):
+        self.file_object = file_object
+
+    @contextmanager
```

Comment:
so here's the trick -- the contextlib.contextmanager decorator is designed to work wrap a generator function all by itself, not as a method on a class.

I think if you simply pulled this out of the class you'd have it :-)